docs/events/meetings.rst

```diff
@@ -1,80 +1,196 @@
+########
 Meetings
-========
+########
+
+This page documents RITlug general meetings.
+Topics include meeting format, meeting topics, how to plan a meeting, and what to do after a meeting.
+Occasionally, there are "special" meetings or project meetings, which may follow a different format (TigerOS meetings, game nights, etc.).
+Special meetings are not covered by this page.
+
+
+*****************
+Meeting checklist
+*****************
+
+Pre-meeting
+===========
+
+#. Plan topic(s) for meeting
+
+#. Announce meeting and topic (see :doc:`../tasks/announcements`)
+
+#. Prepare presentation, slides, or handouts
+
+#. Advertise special topics (e.g. posters)
+
+Meeting
+=======
+
+#. Ensure at least one eboard member is present prior to meeting start time (15 minutes preferably)
+
+#. At start, share any announcements and introduce presenter
+
+#. Ensure at least one eboard member is present until the end of the meeting
+
+Post-meeting
+============
+
+#. Collect slides as PDF from presenter, post to RITlug website
+
+#. If guest speaker, be sure to thank them
+
+#. If no other classes after, notify a lab assistant at The Cage that the room is no longer in use
```

Comment:
Maybe specify that the cage is in the Sys Admin lab for those who are not familiar.